playtime/init.lua

```diff
@@ -53,15 +53,24 @@ local function format_duration(seconds)
 end
 
 minetest.register_chatcommand("playtime", {
-	params = "",
+	params = "<player_name>",
 	description = S("See your total playtime on this server"),
-	func = function(name)
-		if minetest.get_player_by_name(name) then
+	func = function(name, player_name)
+		if player_name == '' then
+            player_name = name
+        end
+		if (player_name == '') and minetest.is_singleplayer() then
+			player_name = 'singleplayer'
+		end
+		if not minetest.player_exists(player_name) then
+			return false, S('You must enter a existing player name!')
+		end
```

Comment:
```suggestion
```

Not needed, the line below does all the checking needed

Reply:
Yes, but this provides more info to the user since this will return if the player actually doesn't exist and the line below returns if they are not online.

Reply:
If a player is online they exist, so it's not really helpful nuance, and many similar commands in CTF don't have it for simplicity/speed reasons

Reply:
I'll check code again, but the intention is that lets say _a is online and you type a_ it will tell you that the player does not exist rather than just saying they are offline.